examples/Cargo.toml

```diff
@@ -75,3 +75,7 @@ path = "clone.rs"
 [[example]]
 name = "speculative-execution"
 path = "speculative-execution.rs"
+
```

Comment:
(I can't attach this comment to the commit message, so I'm placing it here)

"It's a first step towards a macro which gets the value by name straight from a result row." - is it still relevant? If yes, how would you imagine this macro look like?

Reply:
**If** we also decide to add column specification information to every row (which is a waste, but to consider), then it would be even more convenient to extract values by name. It will have a performance penalty though, so I'm not 100% sure if we want to proceed with that direction

Reply:
OK, I see. I don't think a macro would be required in that case, we could just add a method (e.g. `get_by_name`) or even overload the indexing operator to work on strings for maximum ergonomics (see the [`std::ops::Index`](https://doc.rust-lang.org/std/ops/trait.Index.html) trait).

I'm not sure about adding column specification per row either. We don't have to clone it for each row, it could keep it behind an atomic pointer (`Arc`) so the cost is not as tragic as an e.g. allocation.